src/main/resources/flattening/config/ssr_month/env.conf

```diff
@@ -0,0 +1,23 @@
+base_path = "target/test/output"
```

Comment:
I think you can omit this file from the PR.

Reply:
I'm usit it for the tests